source/en/tutorial/anomaly_ruby.rst

```diff
@@ -44,84 +44,86 @@ In this sample program, we will explain 1) how to configure the learning-algorit
 **anomaly.rb**
 
 .. code-block:: ruby
+ :linenos:
```

Comment:
The position aberration occurs original anomaly python tutorial and other tutorials.
- Anomaly: http://jubat.us/ja/tutorial/anomaly_python.html
- Recommender: http://jubat.us/ja/tutorial/recommender_python.html

Classifier tutorials have no position aberration, but it looks no difference between anomaly tutorials and classifier tutorials in source codes.
- Classifier: http://jubat.us/en/tutorial/classifier_python.html

I think this is Jubatus sphinx theme problem, because the  position aberration problem does not occur using default sphinx theme. (Please try to edit source/conf.py Row 98 and compile html.)